db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_12_143158) do
+ActiveRecord::Schema.define(version: 2022_09_06_140208) do
```

Comment:
Only the `trainees` table is updated in this schema, but there is also an update to the `hesa_students` table in the migration.